src/_c_internal_utils.c

```diff
@@ -83,7 +148,7 @@ static PyMethodDef functions[] = {
      "always returns None."},
     {"Win32_SetForegroundWindow",
      (PyCFunction)mpl_SetForegroundWindow, METH_O,
-     "Win32_SetForegroundWindow(hwnd)\n--\n\n"
+     "Win32_SetForegroundWindow(hwnd, /)\n--\n\n"
```

Comment:
What is the slash doing here?

Reply:
This marks `hwnd` as being positional-only (https://www.python.org/dev/peps/pep-0570/), which it is (being implemented with `METH_O`).